app/javascript/modCenter/singleArticle/__tests__/singleArticle.test.jsx

```diff
@@ -46,6 +46,20 @@ describe('<SingleArticle />', () => {
     expect(queryByText(getTestArticle().title)).toBeDefined();
   });
 
+  it('renders the new clickable article title', () => {
+    const { container } = render(
+      <>
+        <SingleArticle {...getTestArticle()} toggleArticle={jest.fn()} />
+        <div
+          data-testid="flag-user-modal"
+          class="flag-user-modal-container hidden"
+        />
+      </>,
+    );
+    const text = getNodeText(container.querySelector('.article-title-link'));
```

Comment:
refactor (non-blocking): You can use preact-testing-library's [getByText](https://testing-library.com/docs/dom-testing-library/api-queries#bytext) instead of using DOM APIs directly.

question: Is the `<div />` in the render necessary for the test?

```jsx
<div
    data-testid="flag-user-modal"
    class="flag-user-modal-container hidden"
/>
```

Reply:
@nickytonline as the OP hasn't responded in a week and as this PR is months old, do you want to finalize this change or shall we merge as is?

Reply:
We can merge it as it's non-blocking.